### ISSUE TITLE
OpenCV for Java with GPU (CUDA) Support

### ISSUE THREAD

**Issue by archenroot** [bytedeco/javacpp-presets#364]
Hi,

first of all thanks for this great contribution!

I am going to process 4 camera system with OpenCV on FullHD resolution with 60FPS in realtime and I am looking for CUDA support. I am on Gentoo system, the openCV library is media-libs/opencv-3.1.0-r7 and is build with CUDA and Java flags. I can even find opencv-310.jar file on my file system from OpenCV installation, but cannot find GPU related packages, so I think this is not supported in the moment.

Does this project handle this somehow?

I tried to compile current dev branch, but it is failing on:
[ERROR] Plugin org.bytedeco:javacpp:1.3.2-SNAPSHOT or one of its dependencies could not be resolved: Could not find artifact org.bytedeco:javacpp:jar:1.3.2-SNAPSHOT in prokyon-public (http://blackhole.archenroot.org:50001/nexus/content/groups/public/) -> [Help 1]

[This artefact seems to be missing in maven](https://search.maven.org/#search%7Cga%7C1%7Ca%3A%22javacpp%22).

So I wasn't able to compile the source.

**Comment by archenroot** [comment]
Ok, update, I am build javacpp latest snapshot, so 1.3.2 version is available and I able to continue with javacpp-presets itself.

**Comment by archenroot** [comment]
Update, I played a little with pom.xml, but finally got working opencv module alone from master branch.

I also got working the Stitching.java example and tried with --try_use_gpu yes switch on 2 images and it works, but I cannot see any kernel running on GPU core with nvidia-smi utility:
`watch -n 0.0001 nvidia-smi`

But I don't see anything executed on CUDA, any explanation? Is CUDA really used, or is there any other example which I try and confirm it?

Thanks guys for your hard work!
